.github/workflows/daily-nightly-jobs.yml

```diff
@@ -1,8 +1,8 @@
 name: Daily nightly jobs
 on:
-  schedule:
-    - cron: "0 0 * * *" # every day at midnight
-  workflow_dispatch: {}
+  pull_request:
```

Comment:
The tests look good, there is a known issue with the rgw multisite test that is not related to this change. So we can merge after you revert this commit.

Reply:
Done. I have already reverted the commit.